Draconic faiths localization

ElMariuso opened the pull request:
<!--
A basic changelog, goes to patch notes of the next version, so should be as short and informative as possible.
-->
## Changelog:
- Updated the localization of dragon religions, and small reorganization of files.

<!--
Before the merge, we recommend you to do these tests with your branch.
-->
## Tests:
- [ ] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log` except `portrait_decals.cpp:101`
- [ ] The mod takes less than 5.5 GB in the Task Manager (Windows)

<!--
If you need to explain something to testers. Otherwise, delete this part.
-->
# How to test:
- Open the game, and check error.log and changed files

ElMariuso committed:
Initial commit

ElMariuso committed:
Initial commit

ElMariuso committed:
gods names

ElMariuso committed:
gods names

ElMariuso committed:
remove comments

ElMariuso committed:
remove comments

ElMariuso committed:
test

ElMariuso committed:
test

ElMariuso committed:
The Seat of the Pantheon as divine realm

ElMariuso committed:
The Seat of the Pantheon as divine realm

ElMariuso committed:
Cult ---> Wyrmcult

ElMariuso committed:
Cult ---> Wyrmcult

Grekopithikos reviewed (dismissed):
LGTM!

ElMariuso committed:
muda da

ElMariuso committed:
muda da

Grekopithikos reviewed (dismissed):
LGTM for real this time

edmund-h commented on localization/english/religion/wc_religion_draconism_l_english.yml:
```diff
@@ -0,0 +1,207 @@
+ draconism_religion_adj:0 "Draconic"
+ draconism_religion_adherent:0 "Draconist"
+ draconism_religion_adherent_plural:0 "Draconists"
+ draconism_religion_desc:0 "Ordained by the Titans to safeguard Azeroth, the Keepers eventually fell from grace. They entrusted their unfinished mission to the Dragon Aspects, who, blessed by the Titans, took up the mantle to protect the world. Many creatures have devoted themselves to serving these mighty beings. When dragons empower mortals, they transform into dragonspawn and drakonid, drawing ...
```
```suggestion
 draconism_religion_desc:0 "The divine mantle of Azeroth's protectorship was passed down first from the Titans to the Keepers, then from the Keepers to the Dragons. For the Dragons themselvses, Draconism exemplifies their race's promise to uphold their sacred duty; yet many lesser creatures have also devoted themselves to serving these mighty beings with religious fervor. When dragons empower these mortals, they transform into dragonspawn and drakonid, drawing their strength from the dragons they serve. "
```

edmund-h commented on localization/english/religion/wc_religion_draconism_l_english.yml:
```diff
@@ -0,0 +1,207 @@
+ alexstraszism_adj:0 "Alexstraszic"
+ alexstraszism_adherent:0 "Alexstraszist"
+ alexstraszism_adherent_plural:0 "Alexstraszists"
+ alexstraszism_desc:0 "The followers of the Red Dragonflight are fierce defenders of life and upholders of order. They believe that the Aspect of Life entrusted the red dragons with the duty of safeguarding all life and administering justice. Their faith emphasizes righteousness and empathy, as they strive to protect the vulnerable from the strong. The red dragons embody the power of the flame, a ...
```
```suggestion
 alexstraszism_desc:0 "The mortal followers of the Red Dragonflight are fierce defenders of life and the order of nature. They believe that the duty of safeguarding all life and administering justice entrusted to the red dragons by the Aspect of Life constitues the truest claim on Divinity through the power of the flame. Their faith emphasizes righteousness and empathy, as they strive to protect the vulnerable from the strong. For the followers of the Red Dragonflight, life is a precious gift that must be cherished and safeguarded. For the Dragons themselves, their belief in and loyalty to the Queen Mother is synonymous with their mission to protect life and justice in keeping with the sacred mission passed down by the Titans of old."
```

edmund-h commented on localization/english/religion/wc_religion_draconism_l_english.yml:
```diff
@@ -0,0 +1,207 @@
+ yserism_adj:0 "Yseric"
+ yserism_adherent:0 "Yserist"
+ yserism_adherent_plural:0 "Yserists"
+ yserism_desc:0 "The followers of the Green Dragonflight are the ancient guardians of nature and champions of balance. They believe that the Aspect of Nature tasked the green dragons with the duty of protecting the natural world and maintaining the delicate equilibrium between all living things. Their faith emphasizes the intertwining of all life and living in harmony with nature. The green drago ...
```
```suggestion
 yserism_desc:0 "The mortal followers of the Green Dragonflight revere the ancient Draconic guardians of nature, champions of balance and growth. The Aspect of Nature entrusted the Green Dragons with the duty of protecting the natural world and maintaining the delicate equilibrium between all living things, and these mortals believe that this constitutes the highest claim on Divinity. For the Dragons themselves, their faith lies in the divine compact they made with the Titans and Keepers. It is manifested through their connection with the sacred Emerald Dream, a realm where the essence of nature resides in the combined subconscious of all living things. While the Green Dragons themselves spend most of their lives asleep, their true power manifests in the Emerald Dream and they often communicate to chosen followers from beyond the veil of sleep."
```

edmund-h commented on localization/english/religion/wc_religion_draconism_l_english.yml:
```diff
@@ -0,0 +1,207 @@
+ malygosism_adj:0 "Malygosic"
+ malygosism_adherent:0 "Malygosist"
+ malygosism_adherent_plural:0 "Malygosists"
+ malygosism_desc:0 "The followers of the Blue Dragonflight are the majestic guardians of the arcane and the keepers of magic. They believe that the wise Aspect of Magic entrusted the blue dragons with the duty of protecting the magical energies of the world and promoting responsible use of magic. Their faith emphasizes understanding and wielding the power of the arcane for the greater good. The b ...
```
```suggestion
 malygosism_desc:0 "The followers of the Blue Dragonflight revere the Blue Dragons as majestic guardians of the arcane and the keepers of magic. They believe that the wise Aspect of Magic granted the Blue Dragons the highest claim on Divinity by entrusting them with protection of the boundless magical energies of Azeroth. Their faith emphasizes understanding and responsibly wielding the power of the arcane for the greater good. The Blue Dragons themselves draw their power from the Nexus, where the leylines converge. For these mysterious and often aloof beings, their duty to maintain the flow of magic and catalog all knowledge is a sacred duty to the ancient Titans."
```

edmund-h commented on localization/english/religion/wc_religion_draconism_l_english.yml:
```diff
@@ -0,0 +1,207 @@
+ nozdormism_adj:0 "Nozdormic"
+ nozdormism_adherent:0 "Nozdormist"
+ nozdormism_adherent_plural:0 "Nozdormists"
+ nozdormism_desc:0 "The followers of the Bronze Dragonflight are the guardians of time and the keepers of history. They believe that the flow of time is sacred, and that the wise Aspect of Time has entrusted the bronze dragons with the duty of safeguarding it. They seek to understand the complex threads of destiny and to protect the past, present, and future from any attempt to alter the timeline ...
```
```suggestion
 nozdormism_desc:0 "Of all the Dragons, the Bronze Dragonflight is most poorly understood, and rightly so. Their domain, which they view as a sacred mission entrusted to them by the Aspect of Time, is the safeguarding of chronological and causal order and the integrity of the underlying mechanics of the universe. This puts the Bronze Dragons far beyond the understanding of even the most intelligent mortals, despite being broadly seen as the friendliest and most convivial with lesser beings on the rare occasion they are able to meet one. Thus, mortal followers of the Bronze Dragonflight take their inscrutable utterances and mysterious doings as proof of their ultimate claim on Divinity through the ancient Titans. Because of their poor understanding of their living deities, mortal followers of the Bronze Dragonflight are strong believers in fate and predestination, seeing themselves as being at the mercy of an inscrutable cosmos."
```

edmund-h committed:
Update descriptions of Draconist faiths

My goal was to provide an explanation of each faith not only from the perspectives of the Dragons themselves, but from that of the mortals who worship them as well.

ElMariuso committed:
Merge branch 'feat/l10n/draconism-loc-update' of https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2 into feat/l10n/draconism-loc-update

Grekopithikos commented on localization/english/religion/wc_religion_draconism_l_english.yml:
```diff
@@ -0,0 +1,207 @@
+ draconism_religion_adj:0 "Draconic"
+ draconism_religion_adherent:0 "Draconist"
+ draconism_religion_adherent_plural:0 "Draconists"
+ draconism_religion_desc:0 "The divine mantle of Azeroth's protectorship was passed down first from the Titans to the Keepers, then from the Keepers to the Dragons. For the Dragons themselvses, Draconism exemplifies their race's promise to uphold their sacred duty; yet many lesser creatures have also devoted themselves to serving these mighty beings with religious fervor. When dragons empower the ...
```
Minor spelling mistake: themselvses

ElMariuso replied:
Fixed

ElMariuso committed:
minor spelling mistake: I win

ElMariuso committed:
uppercase fix

Grekopithikos reviewed (approved):
LGTM for real real this time
